File: CleanFix/WebApi/Entidades/Company.cs
namespace WebApi.Entidades;

public class Company
{
    public int Id { get; set; } // Identificador único de la empresa
    public string? Name { get; set; }
    public string? Address { get; set; }
    public string? Number { get; set; }
    public string? Email { get; set; }
    public IssueType? Type { get; set; } // Ejemplo: "Electricidad", "Fontanería", etc.
    public decimal? Price { get; set; }
    public int? WorkTime { get; set; }
}

public enum IssueType
{
    Plumbing,
    Electrical,
    Carpentry,
    Painting,
    Flooring,
    Cleaning,
    Ready
}

File: CleanFix-frontend/src/app/core/domain/models/Company.ts
export interface Company {
  id: number
  name?: string
  address?: string
  number?: string
  email?: string
  IssueType?: IssueType
  price?: number
  workTime?: number
}

export enum IssueType {
  Fontaneria = 'Fontanería',
  Electricidad = 'Electricidad',
  Carpinteria = 'Carpintería',
  Pintura = 'Pintura',
  Suelos = 'Suelos',
  Limpieza = 'Limpieza',
  Listo = 'Listo',
}
